packages/in_app_purchase/in_app_purchase_storekit/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: in_app_purchase_storekit
 description: An implementation for the iOS and macOS platforms of the Flutter `in_app_purchase` plugin. This uses the StoreKit Framework.
 repository: https://github.com/flutter/plugins/tree/main/packages/in_app_purchase/in_app_purchase_storekit
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+in_app_purchase%22
-version: 0.3.4
+version: 0.3.5
 
 environment:
   sdk: ">=2.14.0 <3.0.0"
```

Comment:
The next line (I can't comment there since it's outside the context) is missing the flutter minimum update to 3.3.

Reply:
🤦‍♀️ this was hard to keep all these package changes organized, thanks for catching.  Glad I split it into reviewable-sized changes.